agents/property.json.example

```diff
@@ -0,0 +1,461 @@
+{
```

Comment:
So what's the different between this one and the `perperty.json.graphs_examples`? Keep one of them is enough?

Reply:
agree

Reply:
delete erperty.json.graphs_examples